src/main.rs

```diff
@@ -209,10 +196,9 @@ fn build_config(version: &str) -> Config {
     opts.optopt("", "auth-server", "change the auth server", "URL");
     opts.optopt("", "auth-client-id", "change the auth client id", "ID");
     opts.optopt("", "auth-client-secret", "change the auth client secret", "SECRET");
-    opts.optopt("", "auth-credentials-file", "change the auth credentials file", "PATH");
     opts.optopt("", "auth-p12-path", "change the auth credentials file", "PATH");
     opts.optopt("", "auth-p12-password", "change the PKCS12 file password", "PASSWORD");
-    opts.optopt("", "auth-expires-in", "change the duration of the access certificate in days", "INT");
+    opts.optopt("", "auth-expiry-days", "change the certificate access duration", "INT");
```

Comment:
Since we're changing the name of the CLI opt and the name of the TOML value, is there a plan to migrate old clients to preserve backwards compatibility?